### PR TITLE
feat(dashboards): Add cols and aggregates to prebuilt widgets

### PR DESCRIPTION
**Pull request by shruthilayaj**

### CODE REVIEW

File: static/app/views/dashboardsV2/data.tsx [hunk 1]
```diff
@@ -232,6 +260,8 @@ export const DASHBOARDS_TEMPLATES: DashboardTemplate[] = [
           {
             name: '',
             fields: ['issue', 'count_unique(user)'],
+            aggregates: ['issue'],
+            columns: ['count_unique(user)'],
```
Comment (edwardgou-sentry) [inline static/app/views/dashboardsV2/data.tsx:264]: I think this is flipped?

File: static/app/views/dashboardsV2/widgetLibrary/data.tsx [hunk 2]
```diff
@@ -71,6 +81,8 @@ export const DEFAULT_WIDGETS: Readonly<Array<WidgetTemplate>> = [
         name: '',
         conditions: '',
         fields: ['count_miserable(user,300)'],
+        aggregates: ['count_miserable(user,300'],
```
Comment (edwardgou-sentry) [inline static/app/views/dashboardsV2/widgetLibrary/data.tsx:84]:
```suggestion
        aggregates: ['count_miserable(user,300)'],
```

**Review by edwardgou-sentry — COMMENTED** [review]
Just a couple typos!

**Review by edwardgou-sentry — APPROVED** [review]
LGTM

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Pull request merged**